src/main/java/wolox/training/models/Book.java

```diff
@@ -81,6 +90,7 @@ public String getTitle() {
     }
 
     public void setTitle(String title) {
+        Preconditions.checkNotNull(title, "Title cannot be null");
```

Comment:
I recommend using a class to store error messages as constants. The best thing would be to create a package called constants and in there host your class of constants for example ErrorConstants.

Reply:
Create class with constants to error messages